src/test/java/com/github/jfasttext/JFastTextTest.java

```diff
@@ -66,15 +66,15 @@ public void test04Predict() throws Exception {
         System.out.printf("\nText: '%s', label: '%s'\n", text, predictedLabel);
     }
 
-	@Test
-	public void test04getArrayVector() throws Exception {
-		JFastText jft = new JFastText();
-		jft.loadModel("src/test/resources/models/supervised.model.bin");
-		String text = "I like soccer";
-		float[] predictedArray = jft.getArrayVector(text);
-		float[] expected = new float[100];
-		assertArrayEquals("", predictedArray, expected, 0.1f);
-	}
+    @Test
```

Comment:
There are a few more whitespace changes left here.

Reply:
I wanted to leave them as whole class use spaces but this one test use tabs, so I think its good time to change this too and make it more consistent.

Reply:
Good point!